### PR TITLE
Used matplotlib Sphinx plot directive to embed output image in the docs.

### PR DESCRIPTION
**Pull request by mbopfNIH**
First fix for #1797

### CODE REVIEW

**Comment by mbopfNIH** [comment]
Comments on this fix:

- Added function call similar to DicomPrintImageTags solution, but could avoid this since there are no arguments passed
- Made assumption of 96dpi display to get plt.imshow to present at "correct" size
- Added extra conf.py parameters to avoid hyperlinks to the source code and images
- The function name get_smiley() isn't "professional", but overloading main() could be confusing. Suggestions?

File: docs/source/conf.py [hunk 1]
```diff
@@ -45,9 +45,14 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx_exec_code',
-    'sphinx_tabs.tabs'
+    'sphinx_tabs.tabs',
+    "matplotlib.sphinxext.plot_directive",
 ]
 
+# Supress hyperlinks in the display of matplotlib.sphinxext.plot_directive plots
```
Comment (dave3d) [inline docs/source/conf.py:52]: suppress (sp)

**Review by dave3d — APPROVED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (mbopfNIH) [reply docs/source/conf.py:52]: Oops. I'll fix it next time I touch that file.

**Review by zivy — COMMENTED** [review]
Generally looks good, but I would aim for a more generic code layout. 

Instead of `create_smiley`, have all examples that return an image use the function signature `main(args)` and the `if __name__==main` pattern. In this particular case the args are ignored. 

The wrapper script then becomes generic, always importing main from the example module (`HelloWorld.py` etc.), forwarding the commandline arguments to the main function and getting the resulting image from it. No need for separate `ExampleX_Wrapper.py` per example.

**Comment by blowekamp** [comment]
@Zivy Generalizing sounds like a good second step. Let's get this case working in RtD, and then generalize it to more examples.

**Pull request merged**